packages/data/feature-flags.ts

```diff
@@ -1,3 +1,5 @@
+import { mainnetStaffs } from 'staffs';
```

Comment:
It should be `./staffs` i guess @sasicodes

Reply:
check the latest commit 🙇🏻